modules/margin-liquidity-pools/src/lib.rs

```diff
@@ -35,7 +41,7 @@ pub trait Trait: frame_system::Trait {
 	type PoolManager: LiquidityPoolManager<LiquidityPoolId, Balance>;
 	type ExistentialDeposit: Get<Balance>;
 	type UpdateOrigin: EnsureOrigin<Self::Origin>;
-	type MaxSwap: Get<Fixed128>;
+	type MaxSwap: Get<SwapRate>;
```

Comment:
I think `MaxSwap` should stay as a `Get<Fixed128>`, it's the cap for long/short rates abs.